packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -63,6 +63,46 @@ const defaultProps = {
   hideAccessoryDisplays: false,
 };
 
+const createConcpetDisplays = (compareConceptDisplays) => {
+  if (compareConceptDisplays.systolic && compareConceptDisplays.diastolic) {
+    if (compareConceptDisplays.systolic === compareConceptDisplays.diastolic) {
+      return <div className={cx('concept-display')}>{compareConceptDisplays.originalSystolic}</div>;
+    }
+    return (
+      <div className={cx('concept-display')}>
+        {compareConceptDisplays.originalSystolic}
+        {' / '}
+        {compareConceptDisplays.originalDiastolic}
+      </div>
+    );
+  }
+  if (compareConceptDisplays.systolic || compareConceptDisplays.diastolic) {
```

Comment:
I think this if can be dropped right? If neither have a value, won't the no data results component be displayed?

Reply:
This is an additional display used for specific situations to add additional context, but not part of the required/standard unit, so a “- -“ may be too much. I will follow up with the difference between the two, how clinicians expect to consume the information.